Kate yoak/sales enablement (#15184)
DataDog/documentation: content/en/partners/sales-enablement.md

---
title: Partner Sales Enablement Guide
kind: documentation
description: "A training roadmap for a new MSP to be ready to lead a Datadog sales engineering process"
private: true
---
## Datadog basics

> First head over to the <a href="https://partners.datadoghq.com/" target="_blank">Partner Portal</a>, log in and click to Training. From there, click to Get Started with the Sales Certification.

Reviewer: ```suggestion
First, head over to the <a href="https://partners.datadoghq.com/" target="_blank">Partner Portal</a>. Log in and click **Training**. Under Sales Certification, click **Get started today**.
```

{{<img src="partners/portal_sales_go.png" alt="Partner Portal Sales Certification" style="max-width:734px">}}

This course walks you through the basics you need to work with your opportunity.

> This course helps you to:

Reviewer: ```suggestion
The course helps you to:
```

- Understand what Datadog is and how to think about it with these modules:
    -  Intro to Monitoring
    -  Product Areas and Use Cases
    -  Product Demo

- Identify an opportunity that's a good fit for Datadog with:
    -  Intro to differentiators
    -  7 signs of technical fit

- Get ready to articulate the message to different audiences with:
    -  Customers and Personas
    -  Elevator pitch training

## Technical demo

> Once you're familiar with the Datadog basics, you should get your Demo Certification.

Author: ```suggestion
Once you have completed the Datadog Sales Certification and are familiar with the Datadog basics, you should get your Demo Certification.
```

> Once you have the idea of the Datadog basics, you should get your Demo Certification.

Reviewer: ```suggestion
Once you're familiar with the Datadog basics, you should get your Demo Certification.
```
Is the Demo Certification part of the Sales Certification? Or are you telling the reader to do the Demo Certification after they've completed the Sales Certification? If it's a separate certification that you complete after you're done with the sales certification, I would say this:
> Once you've completed the Sales Certification and you're familiar with the Datadog basics, you should get your Demo Certification.

{{<img src="partners/portal_demo_go.png" alt="Partner Portal Demo Certification" style="max-width: 367px">}}

Not only does this course teach you to deliver a technical demo, it further helps you understand Datadog's approach to observability and its unified collaborative platform.

## Proof of value

> After giving a technical demo, you will be working with the customer to develop an initial proof of value solution. To prepare for this, obtain the Datadog Technical Certification.

Reviewer: ```suggestion
After giving a technical demo, you'll work with the customer to develop an initial proof of value solution. To prepare for this, obtain the Datadog Technical Certification.
```

{{<img src="partners/portal_tech_go.png" alt="Partner Portal Technical Certification" style="max-width:367px">}}

> This course

Reviewer: ```suggestion
This course:
```

- Explains the basics behind Datadog core products

> - Addresses the market positioning of each

Reviewer: ```suggestion
- Addresses the market positioning of each product
```

- Describes customer success 
- Helps you position individual product offerings

> To guide a customer through a successful proof of value engagement, you need deeper product knowledge. Get started at the Datadog Learning Center with [Datadog Fundamentals Learning Path][1]. The courses include hands-on labs and get you a technical understanding of how Datadog works.

Reviewer: ```suggestion
To guide a customer through a successful proof of value engagement, you need deeper product knowledge. Get started with the [Datadog Fundamentals Learning Path][1] offered by the Datadog Learning Center. The Learning Center courses include hands-on labs and deepen your technical understanding of how Datadog works.
```

## Pricing

In order to complete your quote, you need pricing basics.

> - [Datadog pricing documentation][2] describes how pricing works: units of measure and metering approach at Datadog
> - [Customer plans][3] provides detail into standard end-customer pricing options
> - Finally, to understand discounts offered, check out the Partner Portal pricing tab:
> 
> {{<img src="partners/portal_pricing_go.png" alt="Partner Portal Pricing Info" style="max-width:734px">}}

Reviewer: ```suggestion
- [Datadog's pricing][2] describes how pricing works, including the units of measurement and the metering approach at Datadog.
- [Customer plans][3] provides details about standard end-customer pricing options.
- Finally, to understand discounts offered, check out the Partner Portal pricing tab:

{{<img src="partners/portal_pricing_go.png" alt="Partner Portal Pricing Info" style="max-width:734px">}}
```

## The Datadog sales engineering process

> If you have gone through the recommended training outlined above, you are now ready to go conduct the Datadog sales engineering process:

Reviewer: ```suggestion
If you have gone through the recommended training outlined above, you're ready to conduct the Datadog sales engineering process:
```

- Identify Opportunity
- Qualify Opportunity
- Register Opportunity
- Technical Demo
- Proof of Value
- Configure, Price, Quote

Should you have any questions during your journey, contact partner-support@datadoghq.com 

[1]: https://learn.datadoghq.com/bundles/dd-fundamentals

> [2]: https://docs.datadoghq.com/account_management/billing/pricing/

Reviewer: ```suggestion
[2]: /account_management/billing/pricing/
```
